### PR TITLE
Show last modified

### PR DESCRIPTION
**Pull request by gideonthomas**
Show the last modified date for a project and update the last modified date on every file change

### CODE REVIEW

File: routes/utils.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    method: "PUT",
+    uri: config.publishURL + "/projects/" + projectId,
+    headers: {
+      "Authorization": "token " + token
+    },
+    body: project,
+    json: true
+  }, function(err, response, body) {
+    if(err) {
+      callback(err, 500);
```
Comment (sedge) [inline routes/utils.js]: Log the error and return a generic 500 here

**Comment by sedge** [comment]
r+ w/ 1 thing.

**Comment by gideonthomas** [comment]
I'll wait on https://github.com/mozilla/thimble.webmaker.org/pull/628 to land so that I can refactor this based on it

**Comment by sedge** [comment]
r+

**Pull request merged**